### ISSUE TITLE
Doesn't install TOR

### ISSUE THREAD

**Issue by rdarioc** [mistydemeo/tigerbrew#264]
Hi all,

here is what I get:

---

brew install tor
==> Installing tor dependency: libevent
==> Downloading https://github.com/downloads/libevent/libevent/libevent-2.0.21-stable.tar.gz
###### ################################################################## 100.0%

curl: (35) error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure
Error: Failed to download resource "libevent"
Download failed: https://github.com/downloads/libevent/libevent/libevent-2.0.21-stable.tar.gz

---

Any idea?

All the best
Rosario

**Comment by e-inquirer** [comment]
Hi,

Having a similar problem:

---

````
==> Installing tor dependency: libevent
==> Downloading https://downloads.sourceforge.net/project/levent/libevent/libeve

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "libevent"
Download failed: https://downloads.sourceforge.net/project/levent/libevent/libevent-2.0/libevent-2.0.22-stable.tar.gz
````

---

The config info:

---

````
$ brew --config
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/mistydemeo/tigerbrew.git
HEAD: 08d3bcf8aa82a19e5d2de2b22dc0a717ae6edcc4
Last commit: 6 weeks ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://ia902307.us.archive.org/31/items/tigerbrew
CPU: single-core 32-bit g4e
OS X: 10.4.11-Power Macintosh
Xcode: 2.5
CLT: N/A
GCC-4.0: build 5370
Clang: N/A
X11: N/A
System Ruby: 1.8.2 => /usr/bin/ruby
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
Ruby: /usr/bin/ruby => /Users/workstation
Java: N/A
````

---

Would appreciate any help.

Thanks

**Comment by rdarioc** [comment]
Don't remember how I fixed,  plus I also updated to the 1.5.8

That being said , of course have you checked if the file curl is downloading is there to begin with? 

If so disable any environment proxy variable you have set in your shell and try again. 

---
R.D. Contarino


On April 29, 2016, 03:40, e-inquirer wrote:
Hi, 

Having a similar problem:

==> Installing tor dependency: libevent ==> Downloading https://downloads.sourceforge.net/project/levent/libevent/libeve curl: (22) The requested URL returned error: 404 Not Found Error: Failed to download resource "libevent" Download failed: https://downloads.sourceforge.net/project/levent/libevent/libevent-2.0/libevent-2.0.22-stable.tar.gz 

The config info:

$ brew --config HOMEBREW_VERSION: 0.9.5 ORIGIN: https://github.com/mistydemeo/tigerbrew.git HEAD: 08d3bcf8aa82a19e5d2de2b22dc0a717ae6edcc4 Last commit: 6 weeks ago HOMEBREW_PREFIX: /usr/local HOMEBREW_REPOSITORY: /usr/local HOMEBREW_CELLAR: /usr/local/Cellar HOMEBREW_BOTTLE_DOMAIN: https://ia902307.us.archive.org/31/items/tigerbrew CPU: single-core 32-bit g4e OS X: 10.4.11-Power Macintosh Xcode: 2.5 CLT: N/A GCC-4.0: build 5370 Clang: N/A X11: N/A System Ruby: 1.8.2 => /usr/bin/ruby Perl: /usr/bin/perl Python: /usr/local/bin/python => /Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python Ruby: /usr/bin/ruby => /Users/workstation Java: N/A 

Would appreciate any help.


Thanks

**Comment by mistydemeo** [comment]
Fixed by updating the URL - thanks for the report!
